ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -59,12 +74,12 @@ def __init__(self, field=None, mesh_scoping=None, mesh=None, through_layers=None
 
     @staticmethod
     def _spec():
```

Comment:
I realize an underscore preceding the method means that it is not visible. I just wanted the doc strings to match what was in the visible doc.